src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/OuterApiService.cs

```diff
@@ -87,5 +87,10 @@ public async Task<CreateCohortResponse> CreateCohort(CreateCohortApimRequest req
         {
             return await _outerApiClient.Post<CreateCohortResponse>(new PostCreateCohortRequest { Data = request });
         }
+
+        public async Task<CreatePriorLearningDataResponse> PriorLearningData(long cohortId, long draftApprenticeshipId, CreatePriorLearningDataApimRequest request)
```

Comment:
As before, prefixing the method with "Update" etc. would make it clearer.

Reply:
Thanks Chris, resolved locally